File: capacity-planning-backend-model/src/main/java/com/att/eg/cptl/capacityplanning/backend/model/ForecastVariableDescriptor.java
package com.att.eg.cptl.capacityplanning.backend.model;

import lombok.Data;
import org.springframework.data.mongodb.core.mapping.Document;

@Data
@Document
public class ForecastVariableDescriptor {
  private String variableName;
  private String variableId;
  private String variableUnit;
  private String sheetName;
  private String sheetId;
  private String folderName;
  private String folderId;
  private String searchKey;
  private String variableType;
}

File: src/app/modules/cpt/interfaces/forecast-variable-descriptor.ts
import { Moment } from 'moment';

export interface ForecastVariableDescriptor {
    variableName: string;
    variableId: string;
    variableUnit: string;
    sheetName: string;
    sheetId: string;
    sheetRefId: string;
    folderName: string;
    folderId: string;
    searchKey: string;
    variableType: string;
    startDate: Moment;
    endDate: Moment;
}
